modules/appeals_api/app/swagger/appeals_api/v1/schemas/notice_of_disagreements.rb

```diff
@@ -26,13 +26,27 @@ class NoticeOfDisagreements
               property :veteran do
                 key :type, :object
                 key :description, 'Veteran Object being submitted in appeal'
-                key :required, %i[homeless]
+                key :required, %i[homeless address]
 
                 property :homeless do
                   key :type, :boolean
                   key :example, false
                   key :description, 'Flag if Veteran is homeless'
                 end
+
+                property :address do
+                  key :type, :object
+                  key :description, 'Address of the Veteran'
+                  key :required, %i[zipCode5]
```

Comment:
I think in this case you can add the `key :required, true` directly to zipCode5, but that's purely stylistic

Reply:
@yakschuss FYI - I tried doing that but it doesn't actually do anything other than add the `required` key to the zipCode5 entry. Doesn't even add the red asterisk to show it's required :( So had to keep this `required zipCode5` around regardless.